documentation/src/main/asciidoc/topics/functional_api.adoc

```diff
@@ -301,14 +292,14 @@ for the metadata parameters already provided by the functional map API.
 
 [[_invocation_parameter]]
 == Invocation Parameter
-link:{javadocroot}/org/infinispan/commons/api/functional/Param.html[Per-invocation parameters]
+link:{javadocroot}/org/infinispan/functional/Param.html[Per-invocation parameters]
 are applied to regular functional map API calls to
 alter the behaviour of certain aspects. Adding per invocation parameters is
 done using the
-link:{javadocroot}/org/infinispan/commons/api/functional/FunctionalMap.html#withParams-org.infinispan.commons.api.functional.Param...-[`withParams(Param<?>...)`]
+link:{javadocroot}/org/infinispan/functional/FunctionalMap.html#withParams-org.infinispan.commons.api.functional.Param...-[`withParams(Param<?>...)`]
 method.
 
-link:{javadocroot}/org/infinispan/commons/api/functional/Param.FutureMode.html[`Param.FutureMode`]
+link:{javadocroot}/org/infinispan/functional/Param.FutureMode.html[`Param.FutureMode`]
```

Comment:
FutureMode has been removed. Paragraph and example can be removed.

Reply:
The next paragraph (about `Param.PersistenceMode`) needs to be revisited (package and links need to be updated)